worlds/baylands.world

```diff
@@ -2,15 +2,17 @@
 <sdf version="1.5">
   <world name="default">
     <scene>
+      <ambient>0.95 0.95 0.95 1</ambient>
```

Comment:
Can you explain this change?

Reply:
Yes, that's for including ambient lighting, there was an issue where people were getting weird lighting/shadows on models since ambient wasn't set in the worlds.